cuda_core/cuda/core/experimental/_event.py

```diff
@@ -47,8 +47,25 @@ class Event:
     the last recorded stream.
 
     Events can be used to monitor device's progress, query completion
-    of work up to event's record, and help establish dependencies
-    between GPU work submissions.
+    of work up to event's record, help establish dependencies
+    between GPU work submissions, and record the elapsed time on GPU:
+
+    .. code-block:: python
+
+        # To create events and record the timing:
+        s = Device(0).create_stream()
+        e1 = s.record(options={"enable_timing": True})
+        # ... run some GPU works ...
+        e2 = s.record(options={"enable_timing": True})
+        e2.sync()
+        print(f"time = {e2 - e1}")
```

Comment:
`f"time = {e2 - e1} milliseconds"`
?
Also below.

Reply:
fixed in commit 8aca468